internal/component/s3_release_source.go

```diff
@@ -256,7 +256,7 @@ func (src S3ReleaseSource) UploadRelease(spec Spec, file io.Reader) (Lock, error
 		return Lock{}, err
 	}
 
-	src.logger.Printf("uploading release %q to %s at %q...\n", spec.Name, src.ID(), remotePath)
+	src.logger.Printf("uploading release %q to %s at %q...\n", spec.Name, src.ReleaseSourceConfig.Bucket, remotePath)
```

Comment:
ID should return the bucket name when the ID is not set. We can merge this now, but I think there is something wrong with the s3 release source implementation.